src/bench.ts

```diff
@@ -32,6 +32,8 @@ export default class Bench extends EventTarget {
 
   now = now;
 
+  isAsyncFunction: (fn: Fn) => boolean;
```

Comment:
This PR looks great to me.

But let's remove this option and the dirty check also! Let's only have what mitata does, because tinybench is more likely to be used in a modern environment.

Reply:
Maybe we can only remove the dirty check and keep this option so user can provide other `isAsyncFunction` implement like `now` option?

Reply:
Actually, It'd be good, but from my perspective, it's not reasonable and not that deeply needed! We'll do it if some folks will it, but not now. Let's remove it also, BTW, Thank you so much for your great work.

Reply:
It has removed